custom_components/berlin_transport/manifest.json

```diff
@@ -2,15 +2,15 @@
   "domain": "berlin_transport",
   "name": "Berlin (BVG) and Brandenburg (VBB) transport",
   "codeowners": ["@vas3k"],
-  "config_flow": true,
+  "config_flow": false,
```

Comment:
I've set config_flow false here to make the CI working. Will enable it in the other PR once config_flow is implemented.